src/app/user/create-user/create-user.component.html

```diff
@@ -177,22 +177,26 @@ <h3><span class="glyphicon glyphicon-user"></span>
         <!-- </div> -->
   
         <!-- <div class="form-group"> -->
-          <label for="name"><span class="glyphicon glyphicon-credit-card"></span> Room Rent</label>
-          <input type="text" class="form-control" id="roomRent" required [(ngModel)]="user.payment.roomRent" name="roomRent">
+          <!-- <label for="name"><span class="glyphicon glyphicon-credit-card"></span> Room Rent</label>
+          <input type="text" class="form-control" id="roomRent" required [(ngModel)]="user.payment.roomRent" name="roomRent"> -->
         <!-- </div>       -->
   
         <!-- <div class="form-group"> -->
           
-                  
+          
      
       </div>
-    </mat-tab>    
+      
+    </mat-tab>        
 </mat-tab-group>
 
+  <div *ngIf="!isHostelPaymentReq || selectedBedInfo" class="join-tenant-button">
+    <button type="submit" class="btn btn-success">Join Tenant</button>           
+  </div>
+</form>
+
+
+
 
-<div *ngIf="!isHostelPaymentReq || selectedBedInfo" class="join-tenant-button" >
```

Comment:
Did u verify this button enabled for when checkbox selected for tenent only registration?